diff -Nur gcstar.orig/share/applications/gcstar.desktop gcstar/share/applications/gcstar.desktop
--- gcstar.orig/share/applications/gcstar.desktop	2008-05-26 16:53:36.000000000 +0000
+++ gcstar/share/applications/gcstar.desktop	2008-06-12 13:45:06.000000000 +0000
@@ -8,6 +8,7 @@
 Comment[es]=Organice sus colecciones de películas, juegos, vídeos, libros, música...
 Comment[fr]=Gérer vos collections de films, jeux vidéos, livres, musique,...
 Comment[pt]=Organize as suas colecções de filmes, jogos, vídeos, livros e música...
+Comment[pl]=Katalog kolekcji
 Exec=gcstar %f
 Icon=gcstar
 Terminal=false
