diff -Nur gcstar.orig/bin/gcstar gcstar/bin/gcstar
--- gcstar.orig/bin/gcstar	2008-10-12 16:42:54.000000000 +0200
+++ gcstar/bin/gcstar	2008-10-12 16:47:49.000000000 +0200
@@ -40,17 +40,17 @@
 use Getopt::Long;
 use File::Path;
 use File::Spec;
-use FindBin qw($RealBin);
 use POSIX qw(locale_h);
 
-$ENV{GCS_BIN_DIR} = $RealBin;
-($ENV{GCS_LIB_DIR} = $ENV{GCS_BIN_DIR}) =~ s/bin\/?$/lib\/gcstar/;
-($ENV{GCS_SHARE_DIR} = $ENV{GCS_BIN_DIR}) =~ s/bin\/?$/share\/gcstar/;
-use lib File::Spec->canonpath("$RealBin/../lib/gcstar");
-if (exists $ENV{PAR_TEMP})
-{
-    unshift @INC, "$RealBin/../lib/gcstar";
-}
+use lib qw(/usr/share/gcstar/lib);
+$ENV{GCS_BIN_DIR} = '/usr/bin';
+$ENV{GCS_LIB_DIR} = '/usr/share/gcstar/lib';
+$ENV{GCS_SHARE_DIR} = '/usr/share/gcstar';
+#if (exists $ENV{PAR_TEMP})
+#{
+#    unshift @INC, "$RealBin/../lib/gcstar";
+#}
+
 
 # For win32, redirect all the output to files
 if ($^O =~ /win32/i)
@@ -68,11 +68,9 @@
 my $home = $ENV{'HOME'};
 $home =~ s/\\/\//g if ($^O =~ /win32/i);
 
-$ENV{XDG_CONFIG_HOME} = $RealBin.'/../config' if ($^O =~ /win32/i);
 $ENV{XDG_CONFIG_HOME} = $home.'/.config' if ! exists $ENV{XDG_CONFIG_HOME};
 mkpath $ENV{XDG_CONFIG_HOME};
 
-$ENV{XDG_DATA_HOME} = $RealBin.'/../data' if ($^O =~ /win32/i);
 $ENV{XDG_DATA_HOME} = $home.'/.local/share' if ! exists $ENV{XDG_DATA_HOME};
 mkpath $ENV{XDG_DATA_HOME};
 
diff -Nur gcstar.orig/install gcstar/install
--- gcstar.orig/install	2008-03-05 03:56:00.000000000 +0000
+++ gcstar/install	2008-06-12 13:55:33.000000000 +0000
@@ -197,7 +197,7 @@
 
     #Copying lib
     verbosePrint $lang{InstallCopyDirectory}, $baseDir.'/lib/gcstar';
-    recursiveCopy('lib/gcstar', $baseDir.'/lib/gcstar');
+    recursiveCopy('lib/gcstar', $baseDir.'/share/gcstar/lib');
 
     #Copying share
     verbosePrint $lang{InstallCopyDirectory}, $baseDir.'/share/gcstar';
