--- gcstar/install.orig	2007-01-11 02:37:48.000000000 +0000
+++ gcstar/install	2007-01-11 02:40:39.000000000 +0000
@@ -103,7 +103,7 @@
     }
     
     unlink $baseDir.'/bin/'.$binName;
-    unlink $baseDir.'/man/man1/'.$binName.'.1.gz';
+    unlink $baseDir.'/share/man/man1/'.$binName.'.1.gz';
 }
 
 sub recursiveCopy
@@ -155,9 +155,9 @@
     mkpath $baseDir.'/bin';
     copy 'bin/gcstar', $baseDir.'/bin/'.$binName;
     
-    verbosePrint $lang{InstallCopyDirectory}, $baseDir.'/man/man1';
-    mkpath $baseDir.'/man/man1';
-    my $manPage = "$baseDir/man/man1/$binName.1";
+    verbosePrint $lang{InstallCopyDirectory}, $baseDir.'/share/man/man1';
+    mkpath $baseDir.'/share/man/man1';
+    my $manPage = "$baseDir/share/man/man1/$binName.1";
     copy 'man/gcstar.1', $manPage;
     `gzip -f $manPage 2>&1 >/dev/null`;
     
